onnxruntime/core/providers/xnnpack/nn/resize.cc

```diff
@@ -18,6 +19,11 @@ bool Resize::IsOnnxNodeSupported(const NodeUnit& node_unit,
                                  const GraphViewer& graph_viewer) {
   bool supported = false;
   do {
+    // Internal NHWC domain starts at opset 11
```

Comment:
If the check is required for all layour sensitive ops, Could we put it ahead of `IsOnnxNodeSupported` in the entry function?